src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -612,7 +623,7 @@ std::vector<DWRITE_FONT_AXIS_VALUE> DxFontRenderData::GetAxisVector(const DWRITE
         }
     }
 
-    if (WI_IsFlagClear(axisTagPresence, AxisTagPresence::Weight))
+    if (WI_IsFlagClear(axisTagPresence, AxisTagPresence::Weight) || fontWeight != DWRITE_FONT_WEIGHT::DWRITE_FONT_WEIGHT_NORMAL)

```

Comment:
This is almost certainly wrong, anyone have a better idea for how we can distinguish between 'the user set bold/something else' and 'this is the weight value to use when the font is normal'?